styleSheet.css

```diff
@@ -29,22 +29,81 @@ ul {
   float: left;
 }
 
-.navBar__list>a{
+.navBar__list > a{
   color: white;
   display: block;
   padding: 14px 16px;
   text-align: center;
   text-decoration: none;
 }
 
-.navBar__list>a:hover{
+.navBar__list > a:hover{
   background-color: #111;
 }
 
-.navBar__list--active>a{
+.navBar__list--active > a{
   color: #4CAF50;
 }
+.navBar__list__closeBtn{
+  color: #818181;
+  display: none;
+  padding: 5px 5px 0px 0px;
+  text-align: right;
+  text-decoration: none;
+}
+
+#bodyResume {
+  position: relative;
+  top :5%;
+}
+.navBar__btn{
+  background-color: #111;
+  border: none;
+  color: #fff;
+  display: none;
+  font-size: calc(1em + 3.5vw);
+}
+
+.navBar__sidebarBtn{
+  height: 6.795%;
+  position: fixed;
+  width: 12.08%;
+  left: 0;
+  top: 0;
+  opacity: 0.7;
+}
 
-#bodyResume{
+@media only screen and (max-width: 768px) {
```

Comment:
did you pick up a approach to use in your design? `mobile-first` or `desktop-first`?

Reply:
Do you mean doing one complete version first?

Reply:
I just want to make sure you know what approach you are using

In mobile first, our basic style is dedicated to mobile and we use media queries to upper sizes
e.g.

```css
body {
  background: red;
}

// This applies for >= 600px
@media (min-width: 600px) {
  body {
    background: green;
  }
}
```

But in a desktop fist approach our basic design is desktop and then we use media queries to targe smaller devices

```css
body {
  background: red;
}

//This applies for <= 720px
@media (max-width: 720px) {
  body {
    background: green;
  }
}

```

If you see the main difference is that with mobile first we always use `min-width` and with desktop first `max-width`

Reply:
In that case, the approach I'm using would be `desktop-first`.